src/views/SignTransactionLedger.vue

```diff
@@ -378,9 +371,9 @@ export default class SignTransactionLedger extends Vue {
     }
 
     .data {
-        margin: .5rem 7rem 0;
+        margin: .25rem 3rem 0;
         opacity: 1;
-        text-align: center;
+        color: var(--nimiq-blue);
```

Comment:
Shouldn't the default body color be nimiq-blue? Why is it necessary to specify it here?

Reply:
`nq-text` has `rgba(31,35,72,.7);` set as color.